lametro/templates/legislation.html

```diff
@@ -267,7 +270,7 @@ <h3>Legislation not found</h3>
 
     <script>
 
-        $("#committee-actions").DataTable({
+        $("#committee-actions,#related-bills").DataTable({
```

Comment:
The `committee-actions` id referred to two different tables on the same page, so I changed one of them to `related-bills`. In order to keep it working with datatables, I added that extra id here.